sdk/servicebus/service-bus/src/models.ts

```diff
@@ -40,6 +40,11 @@ export interface InternalMessageHandlers<ReceivedMessageT>
  */
 export type ReceiveMode = "peekLock" | "receiveAndDelete";
 
+/**
+ * Represents the sub queue that is applicable for any queue or subscription.
+ */
+export type SubQueue = "deadLetter" | "transferDeadLetter";
```

Comment:
Can we JSDoc comment the individual string union values?

If so we can point to the dead letter documentation (or maybe even a sample?)

Reply:
Good idea, added in https://github.com/Azure/azure-sdk-for-js/pull/10766/commits/01652085cdb57d66d5d18f8b52d18db51f2f212e